src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -132,6 +132,9 @@ export class BrightScriptDebugSession extends BaseDebugSession {
 
         this.enableDebugProtocol = this.launchConfiguration.enableDebugProtocol;
 
+        //do a DNS lookup for the host to fix issues with roku rejecting ECP
+        this.launchConfiguration.host = await util.dnsLookup(this.launchConfiguration.host);
```

Comment:
Roku will reject ECP commands sent using a hostname, so we need to use the IP address. This PR has started using more home presses, so we need to convert the hostname into an IP address.